scripts/fast-reboot-filter-routes.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+import os
+import utilities_common.cli as clicommon
+import syslog
+import traceback
+import click
+from builtins import str #for unicode conversion in python2
```

Comment:
Looks like you are enforcing the env to be `python3` (top of file). Is Python2 handling still needed?

Reply:
Modified and validated.